policykit/policyengine/models.py

```diff
@@ -983,6 +983,14 @@ class Policy(models.Model):
     modified_at = models.DateTimeField(auto_now=True)
     """Datetime object representing the last time the policy was modified."""
 
+    policy_template = models.OneToOneField(
```

Comment:
does this require a migration file?

Reply:
Yes; I did try to migrate it but could not do it on my instance because the newly added code requires python3.10 which is not that compatible with my ubuntu instance 18.04 --- a huge rabbit hole. wondering whether we could migrate on the production instance after merging the code

Reply:
created  the new migration file